feat: learn english block 6.2 (#52539)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-share-your-progress-in-weekly-stand-up-meetings/6579e102b5a7223f0d0e9fc4.md

---
id: 6579e102b5a7223f0d0e9fc4
title: Task 52
challengeType: 22
dashedName: task-52
audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
---

<!--
AUDIO REFERENCE: 
Bob: "I _ with you. It might help to have a _ style guide for _ language."
-->

# --description--

> The word 'each' refers to every one of a group of items, considered separately. It's often used to emphasize individual attention or distinction within a group. For example, 'Each team member has a specific role' highlights that every person in the team has a unique role.

Reviewer: ```suggestion
The word `each` refers to every one of a group of items, considered separately. It's often used to emphasize individual attention or distinction within a group. For example, `Each team member has a specific role` highlights that every person in the team has a unique role.
```

# --fillInTheBlank--

## --sentence--

`Bob: "I _ with you. It might help to have a _ style guide for _ language."`

## --blanks--

`agree`

### --feedback--

'Agree' shows Bob's concurrence with Sophie's suggestion.

---

`clear`

### --feedback--

> 'Clear' emphasizes the need for an understandable and precise guide.

Reviewer: ```suggestion
`Clear` emphasizes the need for an understandable and precise guide.
```

---

`each`

### --feedback--

> 'Each' refers to every programming language they use, considered separately.

Reviewer: ```suggestion
`Each` refers to every programming language they use, considered separately.
```
